### DISCUSSION TITLE
Zweiter Victron MPPT /VE.Direct nutzbar?

### DISCUSSION THREAD

**Discussion by rs3hc** [hoylabs/OpenDTU-OnBattery#535] [Ideas]
Da ich 2x2 Panels mit unterschiedlicher Ausrichtung habe, wollte ich fragen ob es möglich zwei Victron MPPT zu nutzen um die Batterie zu beladen?

**Comment by Tiese** [comment]
Guten Morgen @rs3hc,
dann will ich dir mal antworten ;-)
Du kannst so viele Laderegler an die Batterie klemmen, wie deine Batterie an Ladestrom verträgt.
Jedoch kann z. Z. nur ein Laderegler von OpenDTU-on Batterie abgefragt werden!

Schöne Grüße
Tiese

**Reply by rs3hc** [reply]
Wäre es nicht evtl möglich, das an einem zweiten ESP "einzusammeln" und am "Master" zu verwerten?

Wäre ja quasi nur zur Auswertung der einzelnen PV-Strings interessant.

Für die Batterie ist es eigentlich egal, da ein Pylontech seine Daten ja per Can schickt und ein Blei-Batteriesatz den Victron Smartshunt für die Daten nutzt. (glaube ich zumindest...)

**Reply by schlimmchen** [reply]
> Wäre ja quasi nur zur Auswertung der einzelnen PV-Strings interessant.

Was ist denn damit gemeint?

Der DPL in OpenDTU-OnBattery kann zur Zeit nur **einen** Victron MPPT berücksichtigen.

**Comment by bikerbiker12** [comment]
Ich denke auch, dass es eine sinnvolle Erweiterung wäre, wenn man mehrere Victron Laderegler einbinden könnte. Aktuell sieht man nur den Ertrag eines LR. Man hat keine Übersicht über den Gesamtertrag aller Strings. Wäre super wenn das klappen würde!

**Comment by Manos1966** [comment]
Ich stelle immer die Frage, nicht um euch zu aergern, sondern weil ich unterschiedliche Meinungen hoeren will.

Ich habe meine Victrons MPPTs noch nicht angeschlossen, daher beschreibe mein (begrenztes) Verstaendniss:
Wenn ich richtig gelesen habe mehrere Victron MPPTs auf die gleiche Batterie ist moeglich, **sie muessen aber Smart sein und miteinander kommunizieren** (per Bluetooth, oder Kabel).
Der Vollständigkeit halber, ein Video diesbezueglich: [Laderegler Parallel anschließen VE.Smart Networking Victron](https://www.youtube.com/watch?v=rkDQcIBGIUA)

Da die Victrons Smart sind, hat man jederzeit Zugriff auf die Daten aller angeschlossenen MPPTs auf die Victron App, richtig?

Warum waere also so besonders sinnvoll, wenn die OpenDTU mehr als einen Victron MPPT einbinden wuerde?
**Nice-to-have** ist es schon, ich frage mich nur ob etwas besonders wichtig noch gibt, dass ich verpasst habe.

**Reply by Sorbat** [reply]
Warum nimmt man mehrere (kleinere?) Laderegler statt einen größeren mit mehr Panels?

**Reply by bikerbiker12** [reply]
Naja, die Anlage wächst und dann kommt halt noch ein LR hinzu! War ja nur eine Frage ob es möglich wäre! Die großen Regler gehen dann oft ins Geld und 2 kleine sind dann günstiger! Das ist alles!

**Reply by Manos1966** [reply]
Ich hoffe aber, du meinst nicht damit die 100/20 
strange behaviour Victron 100/20 with variable cloudiness #513

**Reply by Sorbat** [reply]
Nee ich hatte an nichts spezielles gedacht. 
Ich wollte ja in etwa (ganz grob) was ähnliches aufbauen wie du, nur bin ich noch viel weiter am Anfang.
Mir wurde im discord gesagt, dass der kleinste laderegler den ich für mein Vorhaben brauche der 150/35 ist und damit ich etwas Puffer habe hab ich den nächstgrößeren genommen. 
Ich dachte vielleicht hat es ja gewisse Vorteile mehrere Lr parallel zu schalten statt einen größeren, daher die Frage ✌🏼

Vielleicht mache ich jetzt am
Besten nur noch genau das was du machst, dann führt mich das irgendwann zum Ziel 😄

**Reply by Manos1966** [reply]
🤣 Alles gut!

Das hier nicht vergessen zu benutzen:
[Victron MPPT Calculation Tool](https://mppt.victronenergy.com/#djU7U0NDMDIwMDM1MDAwO2V5SnplWE4wWlcwaU9qUTRMQ0ppWVhSMFpYSjVJam8xTkgwPTtleUpzWVhRaU9qVXdMamN6TnpRekxDSnNibWNpT2pjdU1EazRNakEyT0N3aWJtRnRaU0k2SWtKdmJtNHNJRWRsY20xaGJua2lmUT09O2V5Sm1aV0YwZFhKbGN5STZXMTBzSW5SbGNtMXBibUZzSWpwMGNuVmxMQ0poYkd4dmQwOTJaWEp6YVhwbFpDSTZkSEoxWlgwPTtXM3NpYlc5a2RXeGxJanA3SW5SNWNHVWlPaUpqZFhOMGIyMGlMQ0poY25ScFkyeGxUblZ0WW1WeUlqcHVkV3hzTENKcFRYQndJam94TVM0NE5Dd2lhVlJsYlhCRGIyVm1aaUk2TUM0d05EZ3NJbWx6WXlJNk1USXVOREVzSW0xaGJuVm1ZV04wZFhKbGNpSTZJaUlzSW5CdmQyVnlJam8wTVRBc0luWk5jSEFpT2pNMExqWXNJblpVWlcxd1EyOWxabVlpT2kwd0xqSTNMQ0oyYjJNaU9qUXhMallzSW1sa0lqcHVkV3hzZlN3aVkzVnpkRzl0VFc5a2RXeGxJanB1ZFd4c0xDSnpaWEpwWlhNaU9qTXNJbkJoY21Gc2JHVnNJam95TENKaVlYUjBaWEo1Um14dllYUldiMngwWVdkbElqbzFOQ3dpYldsdVZHVnRjR1Z5WVhSMWNtVWlPaTB5TUN3aWJXRjRWR1Z0Y0dWeVlYUjFjbVVpT2pZMUxDSjJiMngwWVdkbElqb3lOQ3dpWTJGaWJHVk1aVzVuZEdnaU9qRTFMQ0pqWVdKc1pVTnliM056VTJWamRHbHZiaUk2Tm4xZA==)
Ich habe schon die Daten deines 150/35 eingegeben 😉

**Reply by Sorbat** [reply]
Ja den Rechner kenn ich, allerdings kam ich damit letztes Mal schon nicht zurecht.
Ich hatte aber auch den 150|45er gekauft und es sollen 3-6 Trina 335w da dran. Ich denke mir das sollte schon passen, das ist ja jetzt nicht unbedingt eine sehr unübliche Konfiguration 😄

**Reply by Manos1966** [reply]
Oh, da passen viel mehr als 6.
welches Model genau?

**Reply by Sorbat** [reply]
SmartSolar MPPT 150/45
Ich weiß ja nicht was noch kommt und ein wenig Puffer zu haben ist eigentlich immer gut 😄
Das nächste große Ziel wäre n gutes Angebot für ne us5000c zu finden und dann würde ich erstmal versuchen das ganze mit den aktuell 3 vorhandenen Panels einmal zusammen zu wurschteln.
Aber da sind noch so viele Fragezeichen 😅

**Reply by Manos1966** [reply]
Ich meinte die PV Paneele

**Reply by Sorbat** [reply]
Achso sorry. 
Aktuell hab ich 2x neg9r.28 440w und ein DE09R.08W 425w.
Würde bei Bedarf entweder von den neg9r.28 nachkaufen oder neg9rc.27 nehmen. 
Ich weiß nicht ob die alle gleich sein müssen oder ob es reicht wenn die ähnlich sind, aber zumindest 2 gleiche hätte ich ja.

**Reply by Manos1966** [reply]
Hmmm... grosse Paneele

[Victron TSM-445 NEG9R.28](https://mppt.victronenergy.com#djU7U0NDMDIwMDM1MDAwO2V5SnplWE4wWlcwaU9qUTRMQ0ppWVhSMFpYSjVJam8xTkgwPTtleUpzWVhRaU9qVXdMamN6TnpRekxDSnNibWNpT2pjdU1EazRNakEyT0N3aWJtRnRaU0k2SWtKdmJtNHNJRWRsY20xaGJua2lmUT09O2V5Sm1aV0YwZFhKbGN5STZXMTBzSW5SbGNtMXBibUZzSWpwMGNuVmxMQ0poYkd4dmQwOTJaWEp6YVhwbFpDSTZkSEoxWlgwPTtXM3NpYlc5a2RXeGxJanA3SW5SNWNHVWlPaUpqZFhOMGIyMGlMQ0poY25ScFkyeGxUblZ0WW1WeUlqcHVkV3hzTENKcFRYQndJam94TUM0d05Td2lhVlJsYlhCRGIyVm1aaUk2TUM0d05Dd2lhWE5qSWpveE1DNDJOeXdpYldGdWRXWmhZM1IxY21WeUlqb2lJaXdpY0c5M1pYSWlPalEwTUN3aWRrMXdjQ0k2TkRRc0luWlVaVzF3UTI5bFptWWlPaTB3TGpJMExDSjJiMk1pT2pVeUxqSXNJbWxrSWpwdWRXeHNmU3dpWTNWemRHOXRUVzlrZFd4bElqcHVkV3hzTENKelpYSnBaWE1pT2pJc0luQmhjbUZzYkdWc0lqb3pMQ0ppWVhSMFpYSjVSbXh2WVhSV2IyeDBZV2RsSWpvMU5Dd2liV2x1VkdWdGNHVnlZWFIxY21VaU9pMHlNQ3dpYldGNFZHVnRjR1Z5WVhSMWNtVWlPalkxTENKMmIyeDBZV2RsSWpveU5Dd2lZMkZpYkdWTVpXNW5kR2dpT2pFMUxDSmpZV0pzWlVOeWIzTnpVMlZqZEdsdmJpSTZObjFk)

**Reply by spcqike** [reply]
> Nice-to-have ist es schon, ich frage mich nur ob etwas besonders wichtig noch gibt, dass ich verpasst habe.

bekommt openDTU denn die Erzeugungsdaten aller Laderegler, wenn nur einer  angeschlossen ist? Wenn nicht, wie verhält es sich mit Solar-Passtrough? das geht dann ja nicht richtig, oder? 

und dann natürlich für die Enthusiasten, die Auswertung / Analyse. Welcher Laderegler bringt mehr? Ist einer, bzw die dort angeschlossenen Paneele, nicht ideal ausgerichtet? Wie verhält sich die Erzeugung im Laufe des Tages mit wandernden Schatten? aber das ist, wie du sagtest, eher nice-to-have 😄 

btw: du musst nicht zwingend die Daten der MOdule raussuchen und übernehmen, der Calculator kennt sehr viele [Dritthersteller ](https://mppt.victronenergy.com#djU7U0NDMDIwMDM1MDAwO2V5SnplWE4wWlcwaU9qUTRMQ0ppWVhSMFpYSjVJam8xTkgwPTtleUpzWVhRaU9qVXdMamN6TnpRekxDSnNibWNpT2pjdU1EazRNakEyT0N3aWJtRnRaU0k2SWtKdmJtNHNJRWRsY20xaGJua2lmUT09O2V5Sm1aV0YwZFhKbGN5STZXMTBzSW5SbGNtMXBibUZzSWpwMGNuVmxMQ0poYkd4dmQwOTJaWEp6YVhwbFpDSTZkSEoxWlgwPTtXM3NpYlc5a2RXeGxJanA3SW1ObGJHeFVlWEJsSWpvaWJXOXVieUlzSW1sTmNIQWlPakV3TGpBNExDSnBWR1Z0Y0VOdlpXWm1Jam93TGpBMExDSnBaQ0k2SWpVM016WXdOU0lzSW1sell5STZNVEF1TmpFc0lteGhZbVZzSWpvaVZISnBibUVnVTI5c1lYSWdSVzVsY21kNUlEUXlOVmN0TkRKV0lGUlRUUzAwTWpVZ1JFVXdPVkl1TURoWElpd2liV0Z1ZFdaaFkzUjFjbVZ5SWpvaVZISnBibUVnVTI5c1lYSWdSVzVsY21kNUlpd2liMjVVYjNBaU9tWmhiSE5sTENKd2IzZGxjaUk2TkRJMUxDSndka05zYjNWa1NXUWlPaUl5TlRFd01ETWlMQ0owZVhCbElqb2lWRk5OTFRReU5TQkVSVEE1VWk0d09GY2lMQ0oyVFhCd0lqbzBNaTR5TENKMlZHVnRjRU52WldabUlqb3RNQzR5TlN3aWRtOWpJam8xTUM0eWZTd2lZM1Z6ZEc5dFRXOWtkV3hsSWpwdWRXeHNMQ0p6WlhKcFpYTWlPaklzSW5CaGNtRnNiR1ZzSWpvekxDSmlZWFIwWlhKNVJteHZZWFJXYjJ4MFlXZGxJam8xTkN3aWJXbHVWR1Z0Y0dWeVlYUjFjbVVpT2kweU1Dd2liV0Y0VkdWdGNHVnlZWFIxY21VaU9qWTFMQ0oyYjJ4MFlXZGxJam95TkN3aVkyRmliR1ZNWlc1bmRHZ2lPakUxTENKallXSnNaVU55YjNOelUyVmpkR2x2YmlJNk5uMWQ=)Module, einfach in dem Feld suchen

**Reply by Sorbat** [reply]
> Hmmm... grosse Paneele

Was meinst du damit?

**Comment by Manos1966** [comment]
> > Hmmm... grosse Paneele
> 
> Was meinst du damit?

Siehe mein Link. Du kannst selber die Parameter anpassen und das Ergebniss sehen (Min. Temperatur C, Anzahl Paneele in Serie/Parallel usw)

Vorausgesetzt ich habe die Daten korrekt eingegeben, du darfst nicht mehr als ZWEI Paneele pro String.
![Capture](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/ace656b6-ca17-4118-a518-7708e3d34446)
Mit DREI, brauchst du einen 250er
![Capture](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/a55379ba-2f4e-4a7b-bc5f-a982d2e2b803)

**Reply by Sorbat** [reply]
Achso! Mh Joar aber das ist ja jetzt nicht so tragisch. 2x2 ist ja auch gut, und sonst muss halt irgendwann der laderegler doch noch getauscht werden

**Reply by Manos1966** [reply]
Du kannst mehrere Strings haben, du muss nur dein Einkauf paar-weise machen (jeweils zwei Paneele).

**Comment by Manos1966** [comment]
Hallo @spcqike 

> wie verhält es sich mit Solar-Passtrough? das geht dann ja nicht richtig, oder?

Ja! das ist der einzige Grund, den ich bis jetzt gefunden habe, dass **für** die Einbindung mehrere Victrons spricht:
**Durch Einsetzen des Solar-Passthrough, das Max. Ladelimit der Batterie einschränken zu können.**
FRAGE: Man könnte aber auch die max Ladespannung der Victrons so anpassen, dass sie die Spannung der Batterie bei z.B. 90% Ladestand entspricht, damit hätte man das Ergebnis auch erreicht. **Sehr-Unschöne** Lösung, wegen des nicht-optimalen Ladens, aber es wäre möglich, oder?  

> und dann natürlich für die Enthusiasten, die Auswertung / Analyse. Welcher Laderegler bringt mehr?

Bin ich 100% Prozent deiner Meinung, das haette ich auch gern, deswegen ist es fuer mich "nice-to-have", da ohne MQTT und Speicherung der Daten, hilft es nicht.
Ausserdem, **vermute** ich, die "Enthusiasten" haben schon die Lösung mit MQTT Anbindung der Victrons direkt auf HomeAssistant.

Die Frage "warum unterstützt OpenDTUonBattery nicht mehrere Victrons?" wird (**Vermutung** von mir) von "nicht Enthusiasten" gestellt. Als ob dies ein "Show-Stopper" wäre, was meiner Meinung nach nicht ist.

> btw: du musst nicht zwingend die Daten der Module raussuchen und übernehmen, der Calculator kennt sehr viele [Dritthersteller ]

DANKE! 👍 😍
